### PR TITLE
Add location and signature fields

### PR DESCRIPTION
**Pull request by nguyer**
Signed-off-by: Nicko Guyer <nicko.guyer@kaleido.io>

### CODE REVIEW

File: src/tokens/tokens.service.ts [hunk 1]
```diff
@@ -401,6 +401,8 @@ class TokenListener implements EventListener {
 
     const commonData = {
       id: transferId,
+      location: 'address=' + event.address,
```
Comment (eberger727) [inline src/tokens/tokens.service.ts:404]: Make this via URLSearchParams:
```
new URLSearchParams({
      address: event.address
})
```

**Review by eberger727 — CHANGES_REQUESTED** [review]
One Comment

File: src/tokens/tokens.service.ts (hunk 1 above)
Comment (nguyer) [reply src/tokens/tokens.service.ts:404]: If there is more than one will these be comma separated? Or `&` separated like query parameters? They should be comma separated. Also, is the order predictable?

Comment (eberger727) [reply src/tokens/tokens.service.ts:404]: Ah, didn't know the need to be comma separated. This should be fine then

**Review by eberger727 — APPROVED** [review]

**Pull request merged**